db/sql/V1.0__initial_schema.sql

```diff
@@ -0,0 +1,13 @@
+CREATE EXTENSION IF NOT EXISTS pgcrypto;
+
+CREATE TABLE finding_tickets (
+     id UUID PRIMARY KEY DEFAULT gen_random_uuid(),
+     team_id UUID NOT NULL,
+     finding_id UUID NOT NULL,
+     url_tracker TEXT NOT NULL,
+
+     created_at TIMESTAMP WITH TIME ZONE DEFAULT NOW(),
+     updated_at TIMESTAMP WITH TIME ZONE
+);
+
```

Comment:
Perfect, I'm going to remove the index creation but I'm not going to declare any additional index creation because I already have added a unique constraint and according to the official documentation an index for that unique key should be created https://www.postgresql.org/docs/15/indexes-unique.html